services/base.js

```diff
@@ -87,6 +87,15 @@ module.exports = class BaseService {
     return result;
   }
 
+  async invokeHandler(namedParams) {
+    try {
+      return await this.handle(namedParams);
+    } catch (error) {
+      console.log(error);
+      return { message: 'error' };
```

Comment:
Lets not scope creep this PR. I think this is fine for refactoring + adding tests but an improvement we should make here soon is the ability to customise this error based on the exception.